src/main/scala/com/cloudant/clouseau/Main.scala

```diff
@@ -55,5 +55,6 @@ object Main extends App {
   val node = Node(name, cookie, nodeconfig)
 
   ClouseauSupervisor.start(node, config)
+  logger.info("Erlang cookie in use: " + cookie)
```

Comment:
the cookie value is a shared secret, it's not appropriate to print it to screen/log files.